org.adempiere.base/src/org/compiere/util/DisplayType.java

```diff
@@ -1135,6 +1137,8 @@ public static String getDescription (int displayType)
 			return "RadiogroupList";
 		case RecordID:
 			return "RecordID";
+		case RecordUU:
+			return "RecordUU";
```

Comment:
perhaps better as "RecordUUID" instead ? UU is abbreviation of  an abbreviation and doesn't means anything outside of the iDempiere context.